src/Controller/Command/CommandListController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Controller\Command;
+
+use App\Entity\{
+    Command,
+    Menu,
+    Product,
+    Category
+};
+
+use App\ORM\EntityManager;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class CommandListController
+{
+    public function __invoke(Request $request, Container $container)
+    {
+        $em = new EntityManager();
+
+        $p1 = new Product('product 1', 15.0, new Category("cat 1"));
```

Comment:
you should never trust PHP for prices in float.
try that in your console :
```shell
$ php -a
php > var_dump(0.1+0.2 === 0.3);
```

Always store currency values in cents with 3 digits after the separator. like 1000 for 1€.